core/domain/services/admin/events/editor/AdvancedEditorData.php

```diff
@@ -552,14 +558,21 @@ public static function getRelationalData($eventId)
         ];
         // Get the IDs of event datetimes.
         $datetimeIds = $eem_datetime->get_col([[
-            'EVT_ID' => $eventId,
+            'EVT_ID'      => $eventId,
             'DTT_deleted' => ['IN', [true, false]],
         ]]);
         foreach ($datetimeIds as $datetimeId) {
             $GID = self::convertToGlobalId($eem_datetime->item_name(), $datetimeId);
             foreach ($related_models as $key => $model) {
+                $deletedKey = "{$prefixes[$key]}_deleted";
                 // Get the IDs of related entities for the datetime ID.
-                $Ids = $model->get_col([['Datetime.DTT_ID' => $datetimeId]]);
+                $Ids = $model->get_col([
+                    [
+                        'Datetime.DTT_ID' => $datetimeId,
+                        $deletedKey       => ['IN', [true, false]],
```

Comment:
id this necessary now?
the `default_where_conditions` parameter should be covering this now

Reply:
OK, I didn't know that.